beebop/filestore.py

```diff
@@ -227,11 +239,42 @@ def tmp(self, p_hash) -> str:
         os.makedirs(tmp_path, exist_ok=True)
         return str(tmp_path)
 
+    def output_tmp(self, p_hash) -> str:
+        """
+        Generates the path to the full assign output folder.
+
+        :param p_hash: [project hash]
+        :return str: [path to full assign output folder]
+        """
+        path = PurePath(self.tmp(p_hash), p_hash)
+        os.makedirs(path, exist_ok=True)
+        return str(path)
+
+    def partial_query_graph_tmp(self, p_hash) -> str:
+        """
+        :param p_hash: [project hash]
+        :return str: [path to partial query graph]
+        """
+        return str(PurePath(self.output_tmp(p_hash), f"{p_hash}_query.subset"))
+
+    def external_previous_query_clustering_tmp(self, p_hash) -> str:
```

Comment:
What is "previous" query clustering here? The first pass with ref db? Or is that a poppunk term?

Reply:
yup poppunk term.. the function param is called `previous_query_clustering`